mobile/src/components/MobileAppRoute.jsx

```diff
@@ -16,7 +18,7 @@ const MobileAppRoute = requireSetup => (
         <Route path='settings' name='mobile.settings' component={Settings} />}
       </Route>
     </Route>
-    <Route path='onboarding' component={OnBoarding} />
+    <Route path='onboarding' component={OnBoarding} onLeave={() => initBar()} />
```

Comment:
thanks for your answer, wait for another advice and I'll merge this PR if it's ok.